PGM/src/main/java/tc/oc/pgm/commands/MapCommands.java

```diff
@@ -243,16 +243,7 @@ private static BaseComponent mapInfoLabel(String key) {
             }
         }
 
-        int maxPlayers = map.getContext()
-                            .features()
-                            .all(TeamFactory.class)
-                            .mapToInt(TeamFactory::getMaxPlayers)
-                            .sum();
-
-        FreeForAllModule ffam = map.getContext().getModule(FreeForAllModule.class);
-        if(ffam != null) {
-            maxPlayers += ffam.getOptions().maxPlayers;
-        }
+        int maxPlayers = map.getPersistentContext().playerLimits().upperEndpoint();
```

Comment:
Does this make a difference? it used to be a sum of teams + ffa, now it's getting the player limit upper endpoint, wich is basically the sum of all modules that allow more players to join (basically also teams + ffa)